docs/integration-testing.md

```diff
@@ -19,53 +18,66 @@ typically run faster. Sometimes that's not (easily) possible, especially when an
 system that is not easily mocked or stubbed such as cloud-based databases.
 
 Therefore, in many cases writing unit tests is more involved that writing an integration test, for example say we wanted
-to test our implementation of a CosmosDB-backed queue. We would have to mock the behaviour of the CosmosDB API, which -
+to test our implementation of a CosmosDB-backed queue. We would have to mock the behavior of the CosmosDB API, which -
 while certainly possible - can get complicated pretty quickly. Now we still might do that for simpler scenarios, but
 eventually we might want to write an integration test that uses a CosmosDB test instance.
 
 ## Coding Guidelines
 
-An integration test is annotated with `@IntegrationTest`, which causes the test runner to ignore it unless the
-`RUN_INTEGRATION_TEST` environment variable is set to `true`.
+EDC codebase has few annotations to help you write and categorize integration tests. The following are some of the available ones:
```

Comment:
I think more relevant is what runtime components are needed to execute the test (e.g. a storage endpoint). That is a more specific definition

Reply:
done

Reply:
submitted